src/components/DragGroup.js

```diff
@@ -0,0 +1,136 @@
+import Piece from "./Piece";
+import React from "react";
+import { dragDestinationOnBoard } from "./Board";
+import { dragDestinationInPool } from "./Pool";
+
+// This component is mounted each time a drag starts.
+export default function DragGroup({ dispatchGameState, gameState }) {
+  const dragState = gameState.dragState;
+  const isShifting = dragState.isShifting;
+  const draggedPieces = gameState.pieces.filter((piece) =>
+    dragState.pieceIDs.includes(piece.id)
+  );
+
+  // Capture the pointer.
+  const dragGroup = React.useRef(null);
+  React.useEffect(() => {
```

Comment:
1. I think I tried that first. It's possible to call `event.currentTarget.setPointerCapture()` in the `onPointerDown` event handler, but the next thing we'll do is move the piece to the `DragGroup`. React will remove the existing `div` element from the document and discard it. When that happens, pointer capture would be lost.

2. It isn't possible to call `setPointerCapture` directly from `DragGroup()`, because the DOM element doesn't exist yet. React is going to create it after I return `<div id="drag-group" ref={dragGroup} ...>` and _then_ I need to call `setPointerCapture`. `useEffect` handles that; React calls the setup function after the `ref` is set.

Reply:
That makes sense. I vaguely recall running into a similar problem when I implemented dragging from the pool. (I think that's why the current version has a ghost piece left in the pool while the drag is in progress -- because I couldn't remove the div without losing the event.)